README.md

```diff
@@ -92,6 +92,26 @@ end
 
 If the lambda returns a falsy value or raises a `Pundit::UnauthorizedError` the field will resolve to `nil`, if it returns a truthy value, control will be passed to the resolve function. Of course, this can be used with `authorize!` as well.
 
+### Scopes
+
+Pundit scopes are supported by using `scope` in the field definition
+
+```ruby
+field :posts
+  scope
+  resolve ...
+end
+```
+
+By default, this will use the Scope definied in the `UserPolicy`. If you do not want to define a scope inside of the policy, you can also pass a lambda to `scope`. The return value will be passed to `resolve` as first argument.
```

Comment:
> this will use the Scope definied in the `UserPolicy`

As far as I understand [this pundit documentation](https://github.com/elabs/pundit/tree/ac2a25d93ccd044ac3ea777a70ffd5e31db10d3a/#scopes) it does not look into `UserPolicy::Scopes` but in `PostPolicy::Scopes` because posts are indexed.

Reply:
Oops, you're right